app/src/main/java/org/oppia/app/player/state/StateFragment.kt

```diff
@@ -0,0 +1,23 @@
+package org.oppia.app.player.state
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** Fragment that contains displays current state of exploration. */
```

Comment:
Maybe instead:

```kotlin
/** Fragment that represents the current state card of an exploration. */
```

Reply:
NB: This is more related to the ``contains displays`` bit, but I think it can be slightly clarified, too.

Reply:
Resolved and added comment as per @BenHenning suggestion